profile/profile.go

```diff
@@ -0,0 +1,89 @@
+package profile
+
+import (
+	"fmt"
+	"os"
+)
+
+type Profile interface {
+
+	// getOrganizationID() retrieves a user's organization ID.
+	GetOrganizationID() string
+
+	// getProjectID() retrieves the user's project ID.
+	GetProjectID() string
+
+	// getEnvironment() retrieves the application's development environment.
+	GetEnvironment() string
+
+	// setOrganizationID() sets a user's organization ID.
+	SetOrganizationID(string) error
+
+	// setProjectID() sets the user's project ID.
+	SetProjectID(string) error
+
+	// setEnvironment() sets the application's development environment.
+	SetEnvironment(string) error
+}
+
+const (
+	envVarHCPOrganizationID = "HCP_ORGANIZATION_ID"
+
+	envVarHCPProjectID = "HCP_PROJECT_ID"
+
+	envVarHCPEnvironment = "HCP_ENVIRONMENT"
+)
+
+// UserProfile implements the Profile interface.
+type UserProfile struct {
+
+	// hcpOrganizationID is the user's organization ID.
+	hcpOrganizationID string
+
+	// hcpProjectID is the user's project ID.
+	hcpProjectID string
+
+	// hcpEnvironment is the application's development environment.
+	hcpEnvironment string
+}
+
+//TODO: Do we need to include comments here or is it covered since we comment the function definitions in the interface?
+func (p *UserProfile) GetOrganizationID() string {
+	return p.hcpOrganizationID
+}
+
+func (p *UserProfile) GetProjectID() string {
+	return p.hcpProjectID
+}
+
+func (p *UserProfile) GetEnvironment() string {
+	return p.hcpEnvironment
+
+}
+
+func (p *UserProfile) SetOrganizationID(id string) error {
+	err := os.Setenv(envVarHCPOrganizationID, id)
```

Comment:
RE: our earlier discussion, we don't want to overwrite environment variables, only read them. Setting the org ID should only involve setting the value in the UserProfile struct.